src/cpu/gemm/gemm.cpp

```diff
@@ -294,10 +294,10 @@ dnnl_status_t dnnl_gemm_s8s8s32(char transa, char transb, char offsetc, dim_t M,
 
 extern "C" dnnl_status_t DNNL_API dnnl_gemm_bf16bf16f32(char transa,
         char transb, dim_t M, dim_t N, dim_t K, float alpha,
-        const bfloat16_t *A, dim_t lda, const bfloat16_t *B, dim_t ldb,
+        const uint16_t *A, dim_t lda, const uint16_t *B, dim_t ldb,
```

Comment:
This is WA since with bfloat16_t is not available at C API level: https://github.com/openvinotoolkit/oneDNN/blob/6a558fcd183dfa60327267c421031db14020cd30/include/dnnl.h#L3771.